packages/service-library/src/servicelib/rabbitmq_utils.py

```diff
@@ -10,11 +11,38 @@
 from tenacity.wait import wait_fixed
 
 from .logging_utils import log_context
+from .rabbitmq_errors import RPCNamespaceInvalidCharsError, RPCNamespaceTooLongError
 
 log = logging.getLogger(__file__)
 
 
 _MINUTE: Final[int] = 60
+_NAMESPACE_CHAR_LIMIT: Final[int] = 100
+
+REGEX_VALIDATE_RABBIT_QUEUE_NAME: Final[str] = r"^[\w\-\.]{1,255}$"
+
+RPCNamespace = str
+
```

Comment:
why not create a real pydantic model here? you could make it sure it has correct sizes and regex. well I guess there is no pydantic in this library.
so forget that.

Reply:
OK I've replaced these with pedantic models and we have proper validation and tests now

Reply:
hmm but models-library is not in service-lib right?